www/index.html

```diff
@@ -96,7 +96,7 @@
             <article class="view-content">
                 <div id="about" style="display: none;" class="container">
                     <h2 id="contents">
-                        <img src="img/icons/kiwix-60.png" alt="Kiwix icon"/>
+                        <img src="img/icons/kiwix-60.png" id='kiwixLogo' alt="Kiwix icon"/>
```

Comment:
Please use `"` instead of `'` to be consistent
```suggestion
                        <img src="img/icons/kiwix-60.png" id="kiwixLogo" alt="Kiwix icon"/>
```